### DISCUSSION TITLE
Update translations from Lokalise

### DISCUSSION THREAD

**Discussion by AlexGustafsson** [home-assistant/frontend#11384] [Localization] [answered]
I've found that a lot of translations have been missing in the latest stable version of Home Assistant. I joined the Lokalise team and noticed that all the mistakes I had found were already fixed. I proof read as many as I could and tried to fix ones that were lacking.

How often are translations updated, is there a routine for it?

I'm particularly interested in the Swedish translation myself - there are hundreds of new translations, but I suspect other translations may have the same issue.

**Comment by spacegaier** [comment]
@bramkragten @zsarnett @balloob How can we ensure we actually pull in the localize translations for each release? The last one I could find was from 8th of November: https://github.com/home-assistant/frontend/commit/9018d4cc18ce65f2d0e968d1c2341204533884ea

I am aware that by design this should only happen for major HA releases to prevent breaks due to already newer frontend coding changes (target for the next future release that is currently in dev at that time), but then at least we should have pulls for the final frontend build before a major release (plus of course before (and during?) the betas).

**Reply by balloob** [reply]
hm I thought this was automated 🤔

**Reply by balloob** [reply]
Talked to Bram. 

Translations are downloaded automatically when we do a release. They are not automatically updated on the `dev` branch anymore.

https://github.com/home-assistant/frontend/blob/dev/.github/workflows/release.yaml#L38

**Comment by bramkragten** [comment, marked as answer]
Translations are downloaded from Lokalise on every build, so when we release a version it will have the latest translations available.

The dev branch is not being updated automatically anymore, I do this manually once in a while just for the dev experience.

**Reply by spacegaier** [reply]
Every build means that also minor HA releases that contain a new HA frontend version will get updated translations now?

**Reply by bramkragten** [reply]
yes

**Reply by spacegaier** [reply]
@AlexGustafsson If you are on the latest stable HA version (2021.12.9 or newer) then all the translations made before January 10th should be in there (since that was the date of the [last frontend build](https://github.com/home-assistant/frontend/actions/runs/1679999606)). If that is not the case, please clear your browser cache.
